articles/azure/azs-how-resize-disk-ps.md

```diff
@@ -0,0 +1,221 @@
+---
+title: How to resize a disk using PowerShell | UKCloud Ltd
+description: Provides help for resizing a disk using PowerShell on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+reviewer: BaileyLawson
+lastreviewed: 14/03/2019 17:00:00
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Resize a disk
+toc_sub3:
+toc_sub4:
+toc_title: Resize a disk - PowerShell
+toc_fullpath: Users/How To/Resize a disk/azs-how-resize-disk-ps.md
+toc_mdlink: azs-how-resize-disk-ps.md
+---
+
+# How to resize a disk using PowerShell
+
+## Overview
+
+When deploying a new virtual machine from an Azure Marketplace image, the default drive size is often 127GB or less. While it is recommended to add additional disks for tasks such as installing applications and for CPU intensive workloads, you may need to expand the initial drive for purposes such as migrating from a physical PC to the VM or to support applications that must be installed on the OS drive.
+
+This guide shows you how to resize disks on UKCloud for Microsoft Azure.
+
+> [!NOTE]
+> Resizing a disk will cause the virtual machine to restart.
+
+### Intended audience
+
+To complete the steps in this guide you must have the appropriate permissions on the resource you are trying to access.
+
+## Prerequisites
+
+Ensure your PowerShell environment is setup as detailed in [Configure the Azure Stack user's PowerShell environment](azs-how-configure-powershell-users.md).
+
```

Comment:
It feels like maybe you should have Declaring variables as a heading here and then have the Resizing an OS disk and Resizing a data disk as options below that. Otherwise I think that the way this ends up being rendered might look a little odd.